README.md

```diff
@@ -57,6 +57,10 @@ to learn about other command-line flags usable to configure the packaged app.
 To have high-resolution icons used by default for an app/domain, please
 contribute to the [icon repository](https://github.com/nativefier/nativefier-icons)!
 
+### Build Commands Library
```

Comment:
Can you also add a concise mention about the library at the beginning of the README's `Troubleshooting` section?